### PR TITLE
Implement question weight, update/edit and cascade delete features for quiz tasks

### PR DESCRIPTION
**Pull request by pasindur99**
## Purpose

The purpose of this PR is to fix #<issue-number>

Add features for quiz tasks
* Question weight
* Update/edit feature
* Cascade delete for quiz tasks

## Goals

Implement features of question weight and update for quiz tasks.

## Checklist
  - [x] Newly introduced environment variables are added to the hosted dev server 
  - [x] Generated the newly added database entities by running `migrate.sh`

## Test environment
 
macOS, Google Chrome

### CODE REVIEW

File: src/main/kotlin/com/example/routes/tasks/taskRouter.kt [hunk 1]
```diff
@@ -137,7 +137,8 @@ fun Route.tasks(context: DSLContext) {
                         var count = 0
```
Comment (Gravewalker666) [inline src/main/kotlin/com/example/routes/tasks/taskRouter.kt]:
```suggestion
                        var totalPoints = 0
```

File: src/main/kotlin/com/example/routes/tasks/taskRouter.kt [hunk 2]
```diff
@@ -137,7 +137,8 @@ fun Route.tasks(context: DSLContext) {
...
                         for (question in task.details.questions) {
                             val givenAnswer = answers[question.number]
-                            if (givenAnswer == question.correctAnswerKey) count++
+                            if (givenAnswer == question.correctAnswerKey)
+                                count += question.answerWeightKey.toInt()
                         }
                         val score = (count.toDouble()/task.details.questions.size)*100
```
Comment (Gravewalker666) [inline src/main/kotlin/com/example/routes/tasks/taskRouter.kt]: Can't just divide by the number of questions in the list, the weight of the questions should also be taken into account

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/example/routes/tasks/taskRouter.kt (hunk 2 above)
Comment (pasindur99) [reply src/main/kotlin/com/example/routes/tasks/taskRouter.kt]: Fixed with [c2e3db6](https://github.com/we-huddle/server/pull/43/commits/c2e3db6ee51cca4058328af208f9ed8b5a45a00e) 
cc: @Gravewalker666

File: src/main/kotlin/com/example/routes/tasks/taskRouter.kt (hunk 1 above)
Comment (pasindur99) [reply src/main/kotlin/com/example/routes/tasks/taskRouter.kt]: Fixed with [c2e3db6](https://github.com/we-huddle/server/pull/43/commits/c2e3db6ee51cca4058328af208f9ed8b5a45a00e) 
cc: @Gravewalker666

**Review by Gravewalker666 — APPROVED** [review]
Awesome work @pasindur99

**Pull request merged**